sys/include/fmt.h

```diff
@@ -214,10 +214,10 @@ size_t fmt_str(char *out, const char *str);
  *
  * Will convert up to @p n digits. Stops at any non-digit or '\0' character.
  *
- * @param[out]  str  Pointer to string to read from
+ * @param[in]   str  Pointer to string to read from
  * @param[in]   n    Maximum nr of characters to consider
```

Comment:
I think "nr" is a german abbreviation (the English equivalent is "no."). While you at it: can you fix it?